front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -15,35 +15,37 @@ import { TableType, type TimeStopsRow } from './types';
 
 type TimesStopsOutputProps = {
   simulatedTrain: SimulationResponseSuccess;
-  pathProperties: PathPropertiesFormatted;
-  operationalPoints: OperationalPointWithTimeAndSpeed[];
-  selectedTrainSchedule: TrainScheduleResult;
+  trainSummary?: TrainScheduleWithDetails;
+  pathProperties?: PathPropertiesFormatted;
```

Comment:
It seems like we need only `operationalPoints` from `pathProperties`, we could pass only this as a prop

```suggestion
  operationalPoints: OperationalPoint[];
```

Reply:
Done !